sql/src/main/java/io/crate/execution/dsl/package-info.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+/**
+ * <p>
+ *  This package contains classes for the execution domain-specific language (DSL).
+ *  This DSL is used to describe how queries are executed.
+ *  <br />
+ *
+ *  "execution" here refers to the *generic* execution framework used to
+ *  execute DQL or DML statements. (DML restricted to those involving queries - like
+ *  insert from subquery, or update by query).
```

Comment:
Only the DQL parts of the DML can make use of the generic framework. Is this correct?